controller/api.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package controller
+
+// StateServingInfo holds network/auth information needed by a controller.
```

Comment:
Was wondering about the filename, but couldn't suggest a better one.

Reply:
i thought about state.go and ran away quickly :-)